src/components/Frame/Frame.scss

```diff
@@ -28,10 +28,14 @@ $skip-vertical-offset: rem(10px);
   transform: translateX(0%);
 
   @include frame-when-nav-displayed {
-    z-index: 1;
-    top: top-bar-height();
+    z-index: z-index(nav, $fixed-element-stacking-order);
     display: flex;
-    height: calc(100% - #{top-bar-height()});
+
+    &:not(.Navigation-globalTheming) {
+      z-index: 1;
```

Comment:
Why does the navigation, when global theming **is** applied, not need this `z-index: 1` style?

Reply:
Nav needs to fall behind the topbar while global theming is enabled to avoid the topbar box shadow from overlaying the nav